### PR TITLE
chore: #121041 - Security updates 2023-09-07

### PR DESCRIPTION
**Pull request by fmrogers**
### Description
Security updates for EchoComponents

### Before
Screenshot from 2023-09-07 12-08-33

### After
Screenshot from 2023-09-07 12-19-50

- **@equinor/eds-core-react:** Did not update as it's minor version are known to have breaking changes.
- **@equinor/eds-icon:** Did not update as it's minor version are known to have breaking changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,87 +37,87 @@
...
+        "@floating-ui/react": "^0.25.4",
+        "chart.js": "^4.4.0",
+        "jest-environment-jsdom": "^29.6.4",
         "lodash.throttle": "^4.1.1",
-        "react-datepicker": "^4.14.1",
+        "react-datepicker": "^4.16.0",
         "react-sortablejs": "^6.1.4",
         "sortablejs": "^1.15.0",
-        "zustand": "^4.3.8"
+        "zustand": "^4.4.1"
```
Comment (hkot1991) [inline package.json:48]: Shouldn’t zustand be a peer dependency? In the latest echo cli Csaba changes so zustand is provided from echo infield, so I think it should be a peer dependency here  🤔 or maybe that will be part of another pr?

Comment (fmrogers) [reply package.json:48]: @OveBDev i believe @CsabaTheKiss is already in the process of merging Echo Components into the monorepo. But i have no issue converting "zustand" into peer dep. What is the min/max version i should use?

**Review by eiriknysted — APPROVED** [review]
